src/index.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+import React, {PropTypes} from 'react';
+import {Hermes} from './hermes';
+import {createStore, bindActionCreators} from '@streammedev/flux-store';
+import valueReducer, {changeValue} from './actions/value';
```

Comment:
This is by design.  Abstractions like that make it harder to trace where things are coming from.  So in this file, you can see exactly which actions and reducers are being used and how.